lib/screens/theme_screen/widgets/skip_and_next_row.dart

```diff
@@ -21,20 +22,24 @@ class SkipAndNextRow extends StatelessWidget {
             width: 70,
             height: 50,
             child: ElevatedButton(
-              onPressed: () {},
+              onPressed: () {
+                  Navigator.of(context).push(
+                  MaterialPageRoute(builder: (context) => LoginScreen()));
+              },
               child: IconButton(
                 icon: const Icon(Icons.arrow_forward),
-                onPressed: () {},
+                onPressed: () {
+                  Navigator.of(context).push(
+                      MaterialPageRoute(builder: (context) => LoginScreen()));
+                },
```

Comment:
Just use `Icon` widget here

Reply:
It needs to be clickable so IconButton is used to set listener